docs/configure/policy/rules.md

```diff
@@ -36,12 +36,17 @@ The following fields can be edited:
 * `AssigneeEmail`
 * `Notes`
 
-### Sending Slack Notifications
+
+## Integrations
+### Slack Notifications
 If you have attached a Slack installation to your organization, you can use the
 `sendSlackNotification` function to send messages. You can pick which channel
 to send to, or send via a webhook URL. You can also customize the message body
 to add mentions etc.
 
+You can also utilize [Slack incoming webhooks](https://slack.com/help/articles/115005265063-Incoming-webhooks-for-Slack)
```

Comment:
shouldn't this title be on top of the actual example that utilizes the incoming webhook?